templates/careers/hiring-process/index.html

```diff
@@ -31,14 +31,14 @@ <h1>Canonical's hiring process</h1>
     </div>
     <div class="u-fixed-width">
       <div class="p-image-container--cinematic is-cover u-hide--small">
-        {{ image(url="https://assets.ubuntu.com/v1/9acad1b0-department-sales-%402x.jpg",
-                alt="",
-                width="2216",
-                height="834",
-                hi_def=True,
-                loading="auto",
-                attrs={"class": "p-image-container__image"},) | safe
-        }}
+        <iframe width="1120"
+                height="630"
+                src="https://www.youtube.com/embed/mOFgDpjRiO8?si=y9vGn5UhnqDIBVSU"
+                title="YouTube video player"
```

Comment:
We can provide a more useful title here 
```suggestion
                title="Our hiring process at Canonical"
```